gravity.sh

```diff
@@ -348,17 +348,24 @@ gravity_CheckDNSResolutionAvailable() {
     echo -e "  ${CROSS} DNS resolution is currently unavailable"
   fi
 
-  str="Waiting until DNS resolution is available..."
+  str="Waiting 120 seconds if DNS resolution becomes available..."
```

Comment:
This doesn't sound quite right - suggest something more like this


```suggestion
  str="Waiting until DNS resolution is available (Max 120 seconds)..."
```

or 

```suggestion
  str="Waiting up to 120 seconds for DNS resolution..."
```